src/documentation/sdks/python-pipeline-dependencies.md

```diff
@@ -0,0 +1,123 @@
+
+**Note:** Remote workers used for pipeline execution typically have a standard Python 2.7 distribution installation. If your code relies only on standard Python packages, then you probably don't need to do anything on this page.
+
+## <a name="pypi"></a>PyPI Dependencies
+
+If your pipeline uses public packages from the [Python Package Index](https://pypi.python.org/pypi), make these packages available remotely by performing the following steps:
+
+**Note:** If your PyPI package depends on a non-Python package (e.g. a package that requires installation on Linux using the `apt-get install` command), see the [PyPI Dependencies with Non-Python Dependencies](#pypiwithnonpython) section instead.
+
+1. Find out which packages are installed on your machine. Run the following command:
+
+        pip freeze > requirements.txt
+
+    This command creates a `requirements.txt` file that lists all packages that are installed on your machine, regardless of where they were installed from.
+
+2. Edit the `requirements.txt` file and leave only the packages that were installed from PyPI and are used in the workflow source. Delete all packages that are not relevant to your code.
+
+3. Run your pipeline with the following command-line option:
+
+        --requirements_file requirements.txt
+
+    The runner will use the `requirements.txt` file to install your additional dependencies onto the remote workers.
+
+**Important:** Remote workers will install all packages listed in the `requirements.txt` file. Because of this, it's very important that you delete non-PyPI packages from the `requirements.txt` file, as stated in step 2. If you don't remove non-PyPI packages, the remote workers attempt to install packages from sources that are unknown to them, and an error occurs.
+
+
+## <a name="localnonpypi"></a>Local or non-PyPI Dependencies
+
+If your pipeline uses packages that are not available publicly (e.g. packages that you've downloaded from a GitHub repo), make these packages available remotely by performing the following steps:
+
+1. Identify which packages are installed on your machine and are not public. Run the following command:
+
+        pip freeze
+
+    This command lists all packages that are installed on your machine, regardless of where they were installed from.
+
+2. Run your pipeline with the following command-line option:
+
+        --extra_package /path/to/package/package-name
+
+
+## <a name="multfiles"></a>Multiple File Dependencies
+
+Often, your pipeline code spans multiple files. To run your project remotely, you must group these files as a Python package. When remote workers spin up, they look for this kind of package in the staging location and install it. To group your files as a Python package and make it available remotely, perform the following steps:
```

Comment:
`start` instead of `spin up`
`they look for this kind of package in the staging location and install it.` -> `they will install the Python package specified using this method.`